modules/apps/frontend-js/frontend-js-tooltip-support-web/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
 	"@generated": "8ae8dc877032bcba179f5ba5280f23976d8b77cf",
-	"@overrides": {
-	},
+	"@overrides": {},
```

Comment:
This is likely causing CI formatting fails. After https://github.com/liferay/liferay-frontend-projects/pull/550, our linter should no longer do this, right?